examples/01_example_wsiread.ipynb

```diff
@@ -21,20 +21,40 @@
     "id": "p3l08vukKAWo"
```

Comment:
<p><u>_on the right of the Kaggle screen, 3) log in to your Kaggle account.</u></p><p>Add 4) tick "Internet" checkbox under Settings, to enable necessary downloads.</p><p> </p><p><em>The stray underscore was intended (by Mostafa I think) to help set this paragraph in italics. I agree with the italics.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Italics still not right. Move underscore to the end, after "downloads."</p>

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/0a7e3eef1bcb98b1137180859fd0cdeb8769d9e9